config.go

```diff
@@ -128,6 +133,9 @@ func defaultConfig() sequinsConfig {
 				SecretAccessKey: "",
 			},
 		},
+		Datadog: datadogConfig{
+			Bind: "localhost:8200",
```

Comment:
This seems like an unconventional usage of `bind`. Are you creating a socket on this address? Or are you sending packages/datagrams to it?